src/type-layout.md

```diff
@@ -86,9 +86,10 @@ String slices are a UTF-8 representation of characters that have the same layout
 
 ## Tuple Layout
 
-Tuples do not have any guarantees about their layout.
+Tuples have the same layout guarantees a struct with the same fields laid out
+according to the default struct representation.
 
-The exception to this is the unit tuple (`()`) which is guaranteed as a
+The exception to this is the unit tuple (`()`), which is guaranteed as a
 zero-sized type to have a size of 0 and an alignment of 1.
```

Comment:
Does the former statement imply this statement?

Reply:
Not as I wrote it here, but you could certainly argue that it should be changed so it does.